packages/vector_graphics_compiler/lib/vector_graphics_compiler.dart

```diff
@@ -253,3 +258,18 @@ Future<Uint8List> encodeSvg(String input, String filename) async {
   }
   return buffer.done().buffer.asUint8List();
 }
+
+/// Arguement configuration object for encodeSvg function.
+class ArgConfig {
+  /// Initializes the class ArgConfig.
+  ArgConfig(this.xml, this.arg);
+
+  /// The xml string from the file.
+  final String xml;
+
+  /// A command line arguement.
+  final String arg;
```

Comment:
Let's just call this `debugName`. This is meant to be a user-facing string that identifies the source of the original data.

Reply:
fixed.